scripts/getsamplesummary.py

```diff
@@ -1,90 +1 @@
-#!/usr/bin/python
-#
-import sys
-import datetime
-import time
-import glob
-import re
-import socket
-import os
-import select
-from access import db, lims
```

Comment:
I'm a little on the fence about the naming of the "access" module. It seems quite general (potential naming clashes) and not very explicit as to what it is used for.

Perhaps it would be nice to build it into a larger Python package like NGI (scilifelab) under the namespace ``clinical(genomics)``. Otherwise I'd just like to bring the name up for discussion. Perhaps just name it ``cg_access`` to make it explicit that it provides access to resources inside Clinical Genomics.